ui/browser/actions/main-actions.js

```diff
@@ -13,32 +13,32 @@ specific language governing permissions and limitations under the License.
 import * as types from '../constants/action-types';
 
 export function createTab(location) {
-  return { type: types.CREATE_TAB, location };
+  return { type: types.CREATE_TAB, location, instrument: 1 };
```

Comment:
On Thu, Apr 7, 2016 at 5:21 PM, Jordan Santell <notifications@github.com>
wrote:

> In ui/browser/actions/main-actions.js
> <https://github.com/mozilla/tofino/pull/42#discussion_r58965928>:
>
> > @@ -13,32 +13,32 @@ specific language governing permissions and limitations under the License.
> >  import * as types from '../constants/action-types';
> >
> >  export function createTab(location) {
> > -  return { type: types.CREATE_TAB, location };
> > +  return { type: types.CREATE_TAB, location, instrument: 1 };
>
> What's instrument here?
>
I renamed this a few times before settling on shorter.  It's a flag
indicating whether to report ("instrument") these events to our metrics
service (GA for now).

I'd be happy with `shouldInstrument`, or really anything you care to
counter with.

> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla/tofino/pull/42/files/d19a86989b3790eb78e6f6e8c57b8e893ab3d7f0#r58965928>
>